packages/wallets/src/components/DesktopWalletsList/DesktopWalletsList.tsx

```diff
@@ -7,26 +7,19 @@ import { WalletsCard } from '../WalletsCard';
 import './DesktopWalletsList.scss';
 
 const DesktopWalletsList: React.FC = () => {
-    const { data: activeWallet } = useActiveWalletAccount();
-    const { isLoading: isAuthorizeLoading, switchAccount } = useAuthorize();
+    const { data: activeWallet, isLoading: isActiveWalletLoading } = useActiveWalletAccount();
+    const { isLoading: isAuthorizeLoading } = useAuthorize();
     const { isLoading: isCurrencyConfigLoading } = useCurrencyConfig();
 
+    const isLoading = isActiveWalletLoading || isAuthorizeLoading || isCurrencyConfigLoading;
```

Comment:
side note for future PRs: right now we have all those waits, due to the fact, that useActiveWalletAccount returns activeWallet object before its actually fully constructed from multiple API endpoints. In ideal scenario, useActiveWalletAccount shouldn't return active wallet until its fully constructed and thus we shoudln't need to have extra checks like that.